.test-defs/infrastructure-test.yaml

```diff
@@ -0,0 +1,19 @@
+kind: TestDefinition
+metadata:
+  name: infrastructure-test
+spec:
+  owner: gardener-oq@listserv.sap.com
```

Comment:
The owner email should rather point to the maintainer of the test not our central output qualification email